v1/README_Cloud_PubSub_to_Datadog.md

```diff
@@ -0,0 +1,199 @@
+Pub/Sub to Datadog Template
+---
+A pipeline that reads from a Pub/Sub subscription and writes to Datadog's Logs API.
+
+:memo: This is a Google-provided template! Please
+check [Provided templates documentation](https://cloud.google.com/dataflow/docs/guides/templates/provided/pubsub-to-datadog)
+on how to use it without having to build from sources using [Create job from template](https://console.cloud.google.com/dataflow/createjob?template=Cloud_PubSub_to_Datadog).
+
+
+:bulb: This is a generated documentation based
+on [Metadata Annotations](https://github.com/GoogleCloudPlatform/DataflowTemplates#metadata-annotations)
+. Do not change this file directly.
+
+## Parameters
+
+### Required Parameters
+
+* **inputSubscription** (Pub/Sub input subscription): Pub/Sub subscription to read the input from, in the format of 'projects/your-project-id/subscriptions/your-subscription-name' (Example: projects/your-project-id/subscriptions/your-subscription-name).
```

Comment:
Some things we should make sure to document:

- The [default Compute Engine service account](https://cloud.google.com/compute/docs/access/service-accounts#default_service_account) will need the [roles/pubsub.subscriber](https://cloud.google.com/pubsub/docs/access-control#pubsub.subscriber) and [roles/pubsub.viewer](https://cloud.google.com/pubsub/docs/access-control#pubsub.viewer) roles assigned either for the project or the Pub/Sub Subscription

Reply:
Thanks! I feel this is well documented by Dataflow: https://cloud.google.com/dataflow/docs/concepts/security-and-permissions#accessing_pubsub

Reply:
(Let me know if you think we are still lacking any important information there)